deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -1,9 +1,12 @@
 <filter containers.**>
-  @type concat
-  key log
-  multiline_start_regexp "#{ENV['MULTILINE_START_REGEXP']}"
-  separator "#{ENV['CONCAT_SEPARATOR']}"
-  timeout_label @NORMAL
+  @type record_transformer
+  enable_ruby
+  renew_record true
+  <record>
+    log    ${record["log"].split(/[\n\t]+/).map! {|item| JSON.parse(item)["log"]}.join("")}
```

Comment:
just confirming: This won't cause issues if somehow the log coming from `containers.**` doesn't have `log`/`stream`/`time` keys?

Reply:
This is only expecting the `log` key and using that to generate the `stream` and `time` key value pairs from that, so it should be fine.

Reply:
But if the string contained in `log` doesn't have `stream` and `time`, i.e. if the user is not using docker, then this wouldn't work, correct? Maybe we should document somewhere that we only support docker for now?

Reply:
Actually it would work even if the `log` doesn't have `stream` and `time`keys.  Its just that if the key is not present, the JSON.parse value will be empty string for the key.